iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeLoginView.swift

```diff
@@ -176,15 +196,231 @@ struct NativeLoginView: View {
                         .padding(.bottom, 25)
                         
                         Button {
-                            identityFlowLayoutType = .InitializePasswordLessLoginViaOtp
+                            navigate(.StartPasswordLessLogin)
                         } label: {
                             Text("Use One Time Password Instead").frame(minWidth: 150)
                         }
                         .buttonStyle(.bordered)
                         .tint(colorScheme == .dark ? .white : .blue)
                         .zIndex(2.0)
                         
-                    case .InitializePasswordLessLoginViaOtp:
+                        Button {
+                            navigate(.StartPasswordReset)
+                        } label: {
+                            Text("Reset Password").frame(minWidth: 150)
+                        }
+                        .buttonStyle(.bordered)
+                        .tint(colorScheme == .dark ? .white : .blue)
+                        .zIndex(2.0)
+                        
+                        Button {
+                            navigate(.StartRegistration)
+                        } label: {
+                            Text("Register").frame(minWidth: 150)
+                        }
+                        .buttonStyle(.bordered)
+                        .tint(colorScheme == .dark ? .white : .blue)
+                        .zIndex(2.0)
+                        
+                    case .StartRegistration:
+                        TextField("Email", text: $email)
+                            .autocapitalization(.none)
```

Comment:
It looks like these fields have a lot of modifiers in common, could some of them be applied higher up in the hierarchy? or maybe a custom modifier to encapsulate them?

Reply:
I was wanting to add that as well, so I don't need much convincing.  Thanks!  I'll get that up soon.

Reply:
https://github.com/forcedotcom/SalesforceMobileSDK-Templates/pull/414/commits/6c7d1c84be7c442457b5518be4914d5e66ed8499
That looks _so_ much *better*!  Thanks, @bbirman!

Reply:
Maybe there's one thing to mention.  There were some very subtle changes in the top and bottom padding values.  However, I thought it looked better when I made them all consistent across the board.  So, I had padding parameters for the modifiers but then removed them.

Reply:
nice, thanks for trying it out!